packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_splash/blocs/splash_bloc.dart

```diff
@@ -76,7 +76,8 @@ Future<void> _initiateAndRedirect() async {
 
     /// Listen for deep links from the app links service and navigate to the path
     _appLinksService.subscribeToUriLinks((uri) {
-      _router.go(uri.path);
+      final destination = '${uri.path}${uri.hasQuery ? '?${uri.query}' : ''}';
```

Comment:
Please check for already implemented utility function that composes the path along with the query params